polyfill/lib/ecmascript.mjs

```diff
@@ -604,6 +604,18 @@ export const ES = ObjectAssign(ObjectAssign({}, ES2019), {
     let post = parts.length ? `.${parts.join('')}` : '';
     return `${secs}${post}`;
   },
+  AbsoluteToString: (absolute, timeZone) => {
+    const dateTime = timeZone.getDateTimeFor(absolute);
```

Comment:
should this be observably calling `getDateTimeFor` on a `timeZone`, or should it be calling an abstract operation?

Reply:
Good catch, that was something I didn't consider when writing this, but I propose leaving it for now. I'm working on a draft of custom TimeZone implementations (see #300) so it will have to call _something_ observably on a TimeZone, whether here or in another abstract operation, but this code will likely change anyway.

Reply:
What I would hope for is that it does a symbol lookup, for which the default is a method that defers entirely to a slot-reading abstract operation.